distribution/docker/build.gradle

```diff
@@ -47,6 +47,11 @@ dependencies {
   tini 'krallin:tini:0.19.0@tini-amd64'
 }
 
+
+testClusters.all {
+  setting 'xpack.security.enabled', 'false'
+}
```

Comment:
This is not needed, I did a whole CI pass without and it passed.
This project doesn't use test clusters.